src/SLCore.IntegrationTests/FileAnalysisTestsBase.cs

```diff
@@ -133,7 +136,9 @@ private static async Task RunSlCoreFileAnalysis(SLCoreTestRunner slCoreTestRunne
 
     protected static string GetFullPath(string fileToAnalyzeRelativePath)
     {
-        return Path.GetFullPath(fileToAnalyzeRelativePath);
+        var currentDomainBaseDirectory = new DirectoryInfo(AppDomain.CurrentDomain.BaseDirectory);
+        var integrationTestBaseDirectory = currentDomainBaseDirectory.Parent.Parent.Parent.FullName;
```

Comment:
This feels a bit unsafe. Can be replaced maybe by
```
var solutionPath= Path.GetDirectoryName(Path.GetDirectoryName(TestContext.TestDir));
return Path.Combine(solutionPath, @"src\SLCore.IntegrationTests", fileToAnalyzeRelativePath);
```

Reply:
This is hardcoding the integration test project name. I will stick with the AppDomain approach as it seems more stable